pkg/proc/stack.go

```diff
@@ -158,6 +158,9 @@ func (g *G) Stacktrace(depth int, opts StacktraceOptions) ([]Stackframe, error)
 	if err != nil {
 		return nil, err
 	}
+	if it.err != nil {
```

Comment:
I think we aren't doing this deliberately, (*stackIterator).stacktrace already checks if there's an error on the iterator and only reporting it if there are no frames returned, if we want to report it always we should change that instead of adding another check here. But also why?

Reply:
This was done to not hide there error when the full stack could not be unwound, because otherwise it gets hidden. Can likely be removed now.